### DISCUSSION TITLE
Received Account warning

### DISCUSSION THREAD

**Discussion by Raiden186** [thatskymod/Sky-CotL-Scripts#112] [General]
Received this warning, only used candle specific scripts (had Dunkun Langit 0.3.5 and music player as active scripts).would keep an eye on your inboxes
![Screenshot_20240808_195708_Sky 1](https://github.com/user-attachments/assets/c3b4d34c-4d35-4b80-8af8-7f106dce7448)

**Comment by RianaStarspaw** [comment]
This just weird, ive got this on a secondary account ive recently created on a brand new laptop through Steam (new account), never used any scripts or anything. What is going on?

**Reply by TheLibur** [reply]
alot of people on the tgc discord received a warning today even tho they havent hacked like us, it seems like theres some sort of bug going on thats triggering these warnings for regular players, just to be safe tho i recommend playing the game normally for a few days. or you can buy the season's ultimate gift because TGC will kindly ignore any type of infraction you do if you give them money lol
